src/index.js

```diff
@@ -189,6 +189,11 @@ export function createEventLogger(options: EventLoggerOptionsType): Function {
     if (endpoint.transformFunction) {
       result = endpoint.transformFunction.call(undefined, result);
     }
+
+    // if nothing to log at the end - just continue
+    if(isEmptyObject(result)) {
```

Comment:
yep I think that will work. it might need to be a single 'bang' - but my boolean logic is sometimes slow to process. but i like it - I'll try it